zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java

```diff
@@ -269,4 +270,62 @@ public void testDisableParagraphRunButton() throws Exception {
     }
 
   }
+
+  @Test
+  public void testTitleButton() throws Exception {
+    if (!endToEndTestEnabled()) {
+      return;
+    }
+    try {
+      createNewNote();
+
+      waitForParagraph(1, "READY");
+      ZeppelinITUtils.sleep(1000, false);
+
+      String xpathToTile = getParagraphXPath(1) + "//div[contains(@class, 'title')]/div";
+      String xpathToSettingIcon = getParagraphXPath(1) + "//span[@class='icon-settings']";
+
+      collector.checkThat("Before Show Title : The title field contains",
+          driver.findElement(By.xpath(xpathToTile)).getText(),
+          CoreMatchers.equalTo(""));
+      driver.findElement(By.xpath(xpathToSettingIcon)).click();
+      collector.checkThat("Before Show Title : The title option in option panel of paragraph is labeled as  ",
+          driver.findElement(By.xpath(getParagraphXPath(1) + "//a[contains(@ng-click, 'showTitle()')]")).getText(),
```

Comment:
Isn't `//a[contains(@ng-click, 'showTitle()')]` the same as `//ul/li/a[@ng-click='showTitle()']` a bit lower in the code?